builder/entrypoint.sh

```diff
@@ -77,6 +77,7 @@ run_tracee_rules() {
         --cache cache-type=mem \
         --cache mem-cache-size=512 \
         --containers=${CONTAINERS_ENRICHMENT:="0"}\
+        --allow-high-capabilities=${EBPF_ALLOW_HIGH_CAPABILITIES:="0"}\
```

Comment:
Yep, so just add that to tracee-rules too and it's good, the env flag should enable both for a good UX.